modules/compiler/spirv-ll/include/spirv-ll/module.h

```diff
@@ -675,7 +675,7 @@ class Module : public ModuleHeader {
   /// @param[in] id The SPIR-V ID to fetch the value for.
   ///
   /// @return A pointer to the Type or nullptr if not found.
-  llvm::Type *getType(spv::Id id) const;
+  llvm::Type *getLLVMType(spv::Id id) const;
```

Comment:
LLVM seems to often style its own name as `Llvm` when it appears in the middle of a camelcased identifier, like `getLlvmType`, see e.g. `constructLLVMLinkCommand` (old, been around for ages, removed in https://reviews.llvm.org/D130020) vs `constructLlvmLinkCommand` (new, added in https://reviews.llvm.org/D124151). Do we want to do the same?

Reply:
Huh, I didn't expect that. I'm conflicted. We already have a series of `*LLVM*` functions and variables in our codebase; see the results for `[a-z]LLVM` for example. We have no instances of `Llvm` from what I can see.

On the other hand, if it's what LLVM styles itself as, it's probably the right thing to do. Not that LLVM is itself consistent. Maybe that's for historical reasons.

What do you think?

Reply:
I think that's the right call, yep.